pipeline/data/requirements/data.in

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/hplt-project/OpusTrainer.git@6c93fb667726504cad1cf5ab1aba140b99110f9d
```

Comment:
Why are we using this revision? Why not [the PyPi package](https://pypi.org/project/opustrainer/)?

A comment on this file might help if someone wants to update it in the future and doesn't know the reason.

Reply:
it's the latest fixes from main that we need

Reply:
added comment